### PR TITLE
Merge Peter's sprint 3 stuff

### PR DESCRIPTION
**Pull request by pdamianov-dev**

### CODE REVIEW

**Review by samanthainneo99 — APPROVED** [review]
:)

File: test/test_suite.py [hunk 1]
```diff
@@ -1,12 +1,3 @@
-'''
-Created on 10/06/20
-@author:   @arana23 - Aparajita Rana
-Pledge:    I pledge my honor to abide by the Stevens Honor System
-SSW555 - Testing Suite
-Goal: Run all tests for all completed User Stories
-'''
-import unittest
-
```
Comment (ming1in) [inline test/test_suite.py:9]: shouldnt delete this

**Review by ming1in — APPROVED** [review]
Looks good, but should address the comment I left. Also should put new seed files into their own folder within `/test/seeds`

File: test/test_suite.py (hunk 1 above)
Comment (pdamianov-dev) [reply test/test_suite.py:9]: Corrected and put back in

**Review by arana23 — APPROVED** [review]

**Pull request merged**